Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version:5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "EnhancedMediaPlayer",
+    platforms: [
+        .iOS(.v15),
+        .macOS(.v13),
+         .tvOS(.v12),
+         .watchOS(.v5)
```

Comment:
Are we sure we want to make this available on all of these platforms? We haven't tested them, so we can't know it works. Btw, wrong indenting

Reply:
Ditto. For now lets just leave this for `iOS`